packages/application-shell/src/components/application-shell-authenticated/application-shell-authenticated.tsx

```diff
@@ -202,6 +202,13 @@ export const ApplicationShellAuthenticated = (
                     defaultFlags={props.defaultFeatureFlags}
                   >
                     <>
+                      <div
+                        id={'staff-bar'}
+                        css={css`
+                          position: relative;
+                          z-index: 30000;
```

Comment:
Why this value?

Reply:
You are right. Not needed. I added it because dropdown was going behind the top navigation container. I could fix it using ui-kit props.